pom.xml

```diff
@@ -215,6 +215,9 @@
         <module>rocketmq-spring-boot-parent</module>
         <module>rocketmq-spring-boot</module>
         <module>rocketmq-spring-boot-starter</module>
+        <module>rocketmq-client-spring-boot</module>
+        <module>rocketmq-client-spring-boot-parent</module>
+        <module>rocketmq-client-spring-boot-starter</module>
```

Comment:
It seems that the name of the module cannot be well distinguished from the original remoting client. Maybe rename o `rocketmq-grpc-client-spring-boot`?

Reply:
Is there a possibility that we don't emphasize gRPC, since the artifact id of 5.x SDK doesn't contain the keyword "gRPC" either?

Reply:
Does this UserGuide need to be presented to users like a wiki document? If so, it is necessary to consider writing a bilingual version of the document like [this](https://github.com/apache/rocketmq-spring/wiki).

Reply:
> Does this UserGuide need to be presented to users like a wiki document? If so, it is necessary to consider writing a bilingual version of the document like [this](https://github.com/apache/rocketmq-spring/wiki).

Yes, I think we also need a user guide.

Reply:
> Is there a possibility that we don't emphasize gRPC, since the artifact id of 5.x SDK doesn't contain the keyword "gRPC" either?

Or rocketmq-v5-client-spring-boot?

Reply:
@RongtongJin  I agree with you.